### DISCUSSION TITLE
Problems with LDSR

### DISCUSSION THREAD

**Discussion by elchupacabrinski** [Coyote-A/ultimate-upscale-for-automatic1111#61] [Q&A]
Hey, I've been using this extension for quite a while now and I usually use LDSR, as it supposedly gives the best results.
However since around 2 weeks I just can't get it to work anymore. 
No matter how big or small the tile size it ALWAYS runs out of VRAM (some time after downsampling).

Is there something I'm doing wrong, or did the requirements change somehow (I've got 8gb)?

**Comment by AlteredCulture** [comment]
Are you still having a problem
